auth-api/src/auth_api/schemas/user_settings.py

```diff
@@ -24,7 +24,8 @@ class UserSettingsSchema(ma.ModelSchema):  # pylint: disable=too-many-ancestors,
     class Meta:  # pylint: disable=too-few-public-methods
         """Maps all of the User Settings fields to a default schema."""
 
-        fields = ('id', 'label', 'urlorigin', 'urlpath', 'type', 'account_type', 'account_status', 'product_settings')
+        fields = ('id', 'label', 'urlorigin', 'urlpath', 'type', 'account_type', 'account_status', 'product_settings',
```

Comment:
no need to add a new entry, instead pass the new url as product_settings

Reply:
changed